src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,55 @@
-export const Tabs = () => {};
+import React from 'react';
+import classNames from 'classnames';
+import { Tab } from '../../react-app-env';
+
+type Props = {
+  tabs: Tab[];
+  selectedTabId: string;
+  onTabSelected: (tab: Tab) => void;
+};
+
+export const Tabs: React.FC<Props> = (
+  {
+    tabs,
+    selectedTabId,
+    onTabSelected,
+  },
+) => {
+  const isCurrent = tabs.some(tab => tab.id === selectedTabId);
+
+  const findSelectedTab = (findtabs: Tab[]) => (
```

Comment:
findtabs is improper variable name. The name should describe what is stored inside, not what you do gonna do with it

Reply:
I really don't understand how you want me to name it